tests/ide/multiple_errors.rflx

```diff
@@ -141,6 +141,7 @@ package RFLX_Invalid is
                -- error: Undefined variable
                if Undef_Var > 5
                   -- error: Subsequent field F1 referenced
+                  -- not reported as Undef_Var aborts checking this path
```

Comment:
We should rather change the example instead of just adding a note that this error is not reported anymore.

Reply:
Fixed.